package-examples/helloworld-kustomize/README.md

```diff
@@ -15,14 +15,17 @@ patch in it.
 
 Get the example package on to local using `kpt pkg get`
 
+```shell
     $ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/helloworld-kustomize
 
       fetching package /package-examples/helloworld-kustomize from https://github.com/GoogleContainerTools/kpt to helloworld-kustomize
+```
 
 ### View the package contents
 
 List the package contents in a tree structure.
 
+```shell
     $ kpt pkg tree helloworld-kustomize/
```

Comment:
Remove indentation since you have block quotes.

Reply:
done.